measuring/questions/adoption-trend.md

```diff
@@ -0,0 +1,12 @@
+[⬑ back to the overall graph](../use_gqm.md)
+
+# **Question:** What is the InnerSource adoption trend?
+
+This question is something an InnerSource program asks while fostering the adoption of InnerSource.
+Lack of adoption, or stagnant adoption may reveal that it is necessary to rethink the program strategy or investigate impediments.
+
+## Related Metrics
+
+| **Metric** | **How it answers the question** | **Gotchas** |
+| --- | --- | --- |
+| [Number of InnerSource repositories](../metrics/number-of-innersource-repositories.md) | The number of (source code) repositories that are consider InnerSource are directly related to the adoption of the practice. | Depending on how you define _InnerSource repositories_, it can be prone to false positives and/or negatives |
```

Comment:
```suggestion
| [Number of InnerSource repositories](../metrics/number-of-innersource-repositories.md) | The number of (source code) repositories that are directly related to the adoption of the practice. | Depending on how you define _InnerSource repositories_, it can report incorrect results |
```

Here's a little more concise version.

I detected a small grammar issue.